### ISSUE TITLE
Making our own json schedules?

### ISSUE THREAD

**Issue by condorcet01** [bep713/madden-franchise-editor#14]
Hello, sorry, this is probably a stupid question but I was wondering where the schedule jsons are stored on the computer, I have gone through as many of the file folders as I could find and I couldnt find them, because I wanted to see if I could add a custom schedule json.

**Comment by NoahJele** [comment]
If you download the source code from the releases, they're located in the 'schedules' folder in the source code.

**Comment by bep713** [comment]
Yeah the editor doesn't provide an easy way to do it but as @NoahJele said above if you're inclined you can download and build the app from source code and modify one of the JSONs.

The schedule editor is going to need a full facelift with M22 anyway because it 1) doesn't work currently and 2) will be even more broken next year.
